OsmAnd/res/values/strings.xml

```diff
@@ -321,7 +321,7 @@
     <string name="quick_action_edit_actions">Edit actions</string>
     <string name="get_osmand_live">Get OsmAnd Live to unlock all features: Daily map updates with unlimited downloads, all paid and free plugins, Wikipedia, Wikivoyage and much more.</string>
     <string name="unirs_render_descr">Modification of the default style to increase contrast of pedestrian and bicycle roads. Uses legacy Mapnik colors.</string>
-    <string name="shared_string_bookmark">Bookmark</string>
+    <string name="shared_string_bookmark">Add bookmark</string>
```

Comment:
This is a wrong change, shared_string_bookmark implies that it will stay as bookmark in case "add bookmark" fits better it requires new string

Reply:
It is difficult to decide whether to translate it as a noun or verb, the string name gives no further clues either.